packages/edit-site/src/components/template-actions/index.js

```diff
@@ -70,48 +85,44 @@ export default function TemplateActions( {
 	}
 
 	return (
-		<DropdownMenu
-			icon={ moreVertical }
+		<DropdownMenuV2
 			label={ __( 'Actions' ) }
 			className={ className }
 			toggleProps={ toggleProps }
+			// trigger={ <Icon icon={ moreVertical } /> }
+			trigger={ <button>hi</button> }
 		>
-			{ ( { onClose } ) => (
-				<MenuGroup>
-					{ isRemovable && (
-						<>
-							<RenameMenuItem
-								template={ template }
-								onClose={ onClose }
-							/>
-							<MenuItem
-								isDestructive
-								isTertiary
-								onClick={ () => {
-									removeTemplate( template );
-									onRemove?.();
-									onClose();
-								} }
-							>
-								{ __( 'Delete' ) }
-							</MenuItem>
-						</>
-					) }
-					{ isRevertable && (
-						<MenuItem
-							info={ __(
-								'Use the template as supplied by the theme.'
-							) }
+			<DropdownMenuGroupV2>
+				{ isRemovable && (
+					<>
+						<RenameMenuItem
+							template={ template }
+							as={ DropdownMenuItemV2 }
+						/>
+						<DropdownMenuItemV2
 							onClick={ () => {
-								revertAndSaveTemplate();
-								onClose();
+								removeTemplate( template );
+								onRemove?.();
 							} }
 						>
-							{ __( 'Clear customizations' ) }
-						</MenuItem>
-					) }
-				</MenuGroup>
-			) }
-		</DropdownMenu>
+							{ __( 'Delete' ) }
+						</DropdownMenuItemV2>
+					</>
+				) }
+				{ isRevertable && (
+					<DropdownMenuItemV2
+						info={ __(
```

Comment:
`DropdownMenuItemV2` doesn't expose an `info` prop — you'll have to use `children` and add/style that text separately